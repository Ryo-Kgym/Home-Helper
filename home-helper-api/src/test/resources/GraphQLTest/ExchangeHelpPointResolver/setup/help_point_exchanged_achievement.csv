id,user_id,exchange_item_id,exchange_count,exchange_total_point,exchanged_datetime
1,u2,1,1,100,2022-12-25 23:16:18
