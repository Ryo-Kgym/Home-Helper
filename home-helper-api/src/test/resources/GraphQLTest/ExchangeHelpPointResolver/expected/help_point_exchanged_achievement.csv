id,user_id,exchange_item_id,exchange_count,exchange_total_point,exchanged_datetime
1,u2,1,1,100,2022-12-25 23:16:18
3,u1,aaa,1,100,2022-12-04 12:34:56.0
4,u1,bbb,3,300,2022-12-04 12:34:56.0
